pkg/gencmd/util/package.go

```diff
@@ -0,0 +1,76 @@
+package util
+
+import (
+	"path/filepath"
+	"strings"
+
+	"github.com/izumin5210/grapi/pkg/cli"
+	"github.com/izumin5210/grapi/pkg/grapicmd/util/fs"
+	"github.com/pkg/errors"
+	"github.com/serenize/snaker"
+)
+
+type ProtoParams struct {
+	Proto struct {
+		Path    string
+		Package string
+	}
+	PbGo struct {
+		Package    string
+		ImportName string
+	}
+}
+
+func BuildProtoParams(path string, rootDir cli.RootDir, protoOutDir string, pkg string) (out ProtoParams, err error) {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function BuildProtoParams should have comment or be unexported